math-scala/src/main/scala/org/apache/mahout/math/algorithms/regression/OrdinaryLeastSquaresModel.scala

```diff
@@ -55,6 +55,8 @@ class OrdinaryLeastSquares[K] extends LinearRegressorFitter[K] {
     }
 
     var X = drmFeatures
+
+    // TODO : move intercept betas to the first column, not the last column ??
```

Comment:
Start a JIRA, don't introduce a comment into the code in a file you're otherwise not touching ;)